site/layouts/partials/course_features.html

```diff
@@ -1,3 +1,4 @@
+{{ if isset "course_info" .Params }}
```

Comment:
yea Carey's comment about the argument order is correct: https://gohugo.io/functions/isset/#readout

Reply:
Yup, looks correct with the latest commit.  I'm running a test run myself with RC data -> `ocw-to-hugo`@release -> `hugo-course-publisher` at this branch.